schematic/visualization/attributes_explorer.py

```diff
@@ -22,34 +23,39 @@ class AttributesExplorer:
     def __init__(
         self,
         path_to_jsonld: str,
-        data_model_labels: DisplayLabelType,
+        data_model_labels: DisplayLabelType = "class_label",
         data_model_grapher: Optional[DataModelGraph] = None,
         data_model_graph_explorer: Optional[DataModelGraphExplorer] = None,
         parsed_data_model: Optional[dict] = None,
+        graph_data_model: Optional[nx.MultiDiGraph] = None,
+        data_model_graph_pickle: Optional[str] = None,
     ) -> None:
         self.path_to_jsonld = path_to_jsonld
 
         self.jsonld = load_json(self.path_to_jsonld)
+        if graph_data_model is not None:
+            self.graph_data_model = graph_data_model
+        elif data_model_graph_pickle is not None:
+            self.graph_data_model = read_pickle(data_model_graph_pickle)
```

Comment:
Just so I'm reading this correctly, people can either provide the nxMultiDiGraph, or provide a pickle file with this.  Therefore, do you want to validate that the pickle files actually contains a nx.MultiDiGraph?

Reply:
yes, thanks!